llvm/test/CodeGen/RISCV/rvv/rvv-peephole-vmerge-to-vmv.mir

```diff
@@ -15,7 +15,7 @@ body: |
     ; CHECK-NEXT: %avl:gprnox0 = COPY $x1
     ; CHECK-NEXT: %mask:vmv0 = PseudoVMSET_M_B8 %avl, 5 /* e32 */
     ; CHECK-NEXT: $v0 = COPY %mask
-    ; CHECK-NEXT: %x:vr = PseudoVMV_V_V_M1 %false, %true, %avl, 5 /* e32 */, 0 /* tu, mu */
+    ; CHECK-NEXT: %x:vr = PseudoVMV_V_V_M1 %false, %true, %avl, 5 /* e32 */, 1 /* ta, mu */
```

Comment:
Why do we need to have %false as the passthru here?

Reply:
I don't think we need to since elements, I think we can just use the passthru given that all the elements come from true or the tail (passthru). I think this is just a detail from the initial implementation in https://reviews.llvm.org/D133255